packages/sw-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -140,7 +145,23 @@ class RequestWrapper {
     assert.atLeastOne({request});
 
     const response = await this.fetch({request});
-    if (response.ok || response.type === 'opaque') {
+
+    // .ok is true if the response status is 2xx. That's the default condition.
+    let cacheable = response.ok;
+    // If we have any cacheWillUpdate callbacks defined...
+    if (this.callbacks.cacheWillUpdate) {
```

Comment:
Nit: Could this be changed to behaviorCallbacks. Just grokking this code it's not obvious that callbacks relates to behaviors.